code-matlab/shared/datatypes/tc/tc.m

```diff
@@ -1,84 +1,50 @@
 function tc_out = tc(varargin)
 %% TC TUNING CURVE DATATYPE
+%   The tc dataype is a container for tuning curves typically used in experimental
+%   analysis.
+%
 %   tc_out = TC(varargin) is a constructor for the tc (tuning curve) struct.
 %
-% youkitan 2014-10-27
+%   INPUTS:
+%
```

Comment:
- in terms of input, I was going to use the tsd constructor as a template
- in terms of examples, there are workflows in code-matlab/example_workflows but I haven't updated those (they use the "old" versions of equivalent functions/datatype)